lib/archiving.js

```diff
@@ -89,6 +89,16 @@ var api;
 *   URL for 10 minutes. To generate a new URL, call the {@link OpenTok#getArchive OpenTok.getArchive()} or
 *   {@link OpenTok#listArchives OpenTok.listArchives()} method.
 *
+* @property {String} resolution
+*   The resolution to be used for Composed Archiving and Broadcasting. This parameter should not be passed
+*   for Individual archives (doing so will result in an error). The following resolutions are supported:
+*   <ul>
+*     <li>"640x480" -- SD archiving output</li>
+*     <li>"1280x720" -- HD archiving output</li>
+*   </ul>
+*   Note: this property is set only for Composed Archiving and Broadcasting.
+*   See the {@link OpenTok#startArchive OpenTok.startArchive()} method.
+*
```

Comment:
Can we remove the documentation since its not yet a public API?

Reply:
Yeah sure, I just removed it.